docs/src/pages/company/careers/careers.md

```diff
@@ -29,14 +29,35 @@ grow from it.
 
 We're currently looking for help in the following areas:
 
-### [Senior Software Engineer](/company/software-engineer/)
+### Engineering
 
-We are looking for senior software engineers to help support our open-source components, assist the Material-UI community, and grow our enterprise edition.
-
-### [Software Engineer Intern](/company/software-engineer-intern/)
+#### [Software Engineer Intern](/company/software-engineer-intern/)
```

Comment:
```suggestion
#### [Software Engineering Intern](/company/software-engineer-intern/)
```

Reply:
Are you sure? I can't remember ever seeing this as a role title.

Reply:
Both seem to be in common use, but no strong preference – I was simply matching the title to the description, since the latter reads better.

Reply:
Right, looking at glassdoor.com, I could find some job titles with "Engineering", but many more with "Engineer".